file.rb

```diff
@@ -51,6 +52,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rental.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] I would like to recommend you have `book` and `date` as parameters and note that The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association :+1: